pyproject.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-psycopg2 = "^2.9.2"
```

Comment:
I'd still leave the ^ there, it tells poetry that any higher version (until the next major one) is OK.

If you ran `poetry update psycopg2` I think it would do that by default.